nemo/collections/asr/models/label_models.py

```diff
@@ -89,9 +89,15 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         self.cal_labels_occurrence_train = False
         self.labels_occurrence = None
 
+        if 'num_classes' in cfg.decoder:
+            num_classes = cfg.decoder.num_classes
+        else:
+            num_classes = cfg.decoder.params.num_classes  # to pass test
```

Comment:
```suggestion
        num_classes = cfg.decoder.num_classes  if 'num_classes' in cfg.decoder else cfg.decoder.params.num_classes
```

Could probably one line this if preferred. Did the parameter move from `decoder.params.num_classes` to `decoder.num_classes`? If so maybe adding a message `# check for backward compatibility` or something

Reply:
I am looking to remove it and revert the test, since this was done long back. Will address this in another PR.